sympy/concrete/summations.py

```diff
@@ -574,13 +576,16 @@ def eval_sum(f, limits):
 
 
 def eval_sum_direct(expr, limits):
+    from sympy.core import Add
```

Comment:
Just curious:

What issue is motivating these changes?
Is it better to do an import in a function than to use C? 
Is it better to use the full path (sympy.core.add) than partial (sympy.core)?